compose.yaml

```diff
@@ -154,7 +154,7 @@ services:
       # If you update anything here that could affect NLP results, consider updating the
       # task_version of any tasks that use this docker.
       - --model=openai/gpt-oss-120b
-      - --revision=bc75b44b8a2a116a0e4c6659bcd1b7969885f423
+      - --revision=b5c939de8f754692c1647ca79fbf85e8c1e70f8a
```

Comment:
Update to the latest version in hugging face, no big changes as far as I can tell from the commits, but they did adjust the prompt templates. I am not bothering to bump any of the task versions using this model, since I just did that in the last (unreleased) commit due to caching changes. So a good time to squeeze this in.